gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineExactDependencies.java

```diff
@@ -232,14 +231,10 @@ private static String getCompileConfigurationName(SourceSet sourceSet) {
     }
 
     private static Map<String, String> excludeRuleAsMap(ExcludeRule rule) {
-        Builder<String, String> excludeRule = ImmutableMap.builder();
-        if (rule.getGroup() != null) {
```

Comment:
Intellij flags this with `Method 'getGroup' inherits annotation from package org.gradle.api.artifacts, thus 'non-null'`